### PR TITLE
Adding knative eventing 1.0 jobs

### PR DESCRIPTION
**Pull request by devguyio**
As per title

### CODE REVIEW

**Comment by devguyio** [comment]
Waiting for knative-eventing#1488
/hold

File: ci-operator/jobs/openshift/knative-eventing/openshift-knative-eventing-release-v1.0-periodics.yaml [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+  - base_ref: release-v1.0
+    org: openshift
+    path_alias: knative.dev/eventing
+    repo: knative-eventing
+  labels:
+    ci-operator.openshift.io/variant: "47"
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-knative-eventing-release-v1.0-47-e2e-aws-ocp-47-continuous
+  spec:
```
Comment (matzew) [inline ci-operator/jobs/openshift/knative-eventing/openshift-knative-eventing-release-v1.0-periodics.yaml:18]: slack reporter?

Comment (devguyio) [reply ci-operator/jobs/openshift/knative-eventing/openshift-knative-eventing-release-v1.0-periodics.yaml:37]: Done

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**